### PR TITLE
Blog app: add API endpoints

### PR DESCRIPTION
**Pull request by Botlhale-Setou**
**Learning Objectives Reached:**

- Build an API that is RESTful.
- Understand importance of APIs and docs following standards.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -9,4 +9,14 @@
...
+
+  namespace :api do
+    namespace :v1 do
+      resources :users, only: [:index, :show, :create] do
+        resources :posts, only: [:index, :show, :create] do
+          resources :comments, only: [:index, :show, :create]
+        end
+      end
+    end
+  end
```
Comment (tugsem) [inline config/routes.rb]: - [x] Kindly fix this issue. You have the right directory but you don't have users_controller file in your api/v1 folder. 
image

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hello @Botlhale-Setou  and @juanmanuelbyc , 👋 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️linters are passing
✔️correct use of gitflow
✔️professional documentation
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (Botlhale-Setou) [reply config/routes.rb]: Thanks @tugsem 
We've removed the routes we didn't need on the routes.rb file.

**Review by addod19 — APPROVED** [review]
### Hi TEAM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### HIGHLIGHTS
- [ ] Nice job fixing the bug raised in the previous review, app works as expected
Screenshot from 2023-04-22 19-03-26


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**